examples/qml_features/rust/src/invokables.rs

```diff
@@ -64,22 +64,22 @@ impl Default for RustInvokablesRust {
 // ANCHOR: book_invokable_impl
 impl qobject::RustInvokables {
     /// Immutable invokable method that returns the QColor
-    fn load_color(&self) -> Result<QColor, i32> {
+    pub fn load_color(&self) -> Result<QColor, i32> {
         Ok(self.as_qcolor())
```

Comment:
Maybe we want to make `as_qcolor` pub as well?
Just to show how documentation looks like if you also have methods on the Rust struct.

Reply:
done